### ISSUE TITLE
how to use lpips loss in tensorflow?

### ISSUE THREAD

**Issue by Fangkang515** [richzhang/PerceptualSimilarity#55]
Thank you for your significant contribution！

It seems  that the LPIPS loss function can not be used directly in tensorflow to train a neural network. What should I do if I want to use it？

**Comment by richzhang** [comment]
Maybe you can take a look at this repo: https://github.com/alexlee-gk/lpips-tensorflow
